mooncake-integration/store/store_py.cpp

```diff
@@ -304,6 +304,18 @@ PYBIND11_MODULE(store, m) {
                                           local_buffer_size, protocol,
                                           rdma_devices, master_server_addr);
              })
+        .def("setup_with_files",
```

Comment:
Could we add more parameters in setup or use environment variables to complete the file setup?

Reply:
Yes, we can.
The only difference between `setup_with_files` and `setup` is that `setup_with_files` takes a `files` argument instead of the `global_segment_size` argument. We can add the `files` argument as the last argument of `setup` for compatibility. Do you think this is better than adding `setup_with_files`?